guides/resource-provisioning/servicebus-queues.md

```diff
@@ -0,0 +1,138 @@
+# Azure Service Bus Queues
+
+This guide describes how to add Azure Service Bus queues to a microservice running on the FFC Platform Kubernetes cluster.
+
+## Create a microservice Managed Identity
+
+If not alreay configured, [add an Azure Managed Identity to your microservice](managed-identity.md).
+
+## Request creation of Service Bus queues
+
+For each queue required by your microservices, you will need:
+* A Service Bus queue for the main branch deploy
```

Comment:
Similar to above I wouldn't relate a branch to a deployment because away from PRs, there won't be any branches involved.  Essentially you'll need a queue for every environment.

Reply:
Updated language